src/OpenRiaServices.Client/Test/Client.Test/Data/OperationTests.cs

```diff
@@ -439,6 +442,91 @@ public void Bug706066_CancelInCallback()
             EnqueueTestComplete();
         }
 
+
+
+        class ExceptionHandleSynchronizationContext : SynchronizationContext
+        {
+            public Exception LastException { get; set; }
+            public override void Send(SendOrPostCallback d, object state)
+            {
+                if (state is ExceptionDispatchInfo exceptionDispatchInfo)
+                {
+                    LastException = exceptionDispatchInfo.SourceException;
+                }
+
+                base.Send(d, state);
+            }
```

Comment:
1. I suggest that the code is made a bit more general and don't depend on the type of the state parameter.
  The test implementation currenly both records the exception and throws it (but Dispatcher version will not rethrow it). The below suggestion changes that to better match WPF behaviour.

```suggestion
            {
                try
                {
                   base.Send(d, state); // or just "d(state)";
                }
                catch (Exception ex)
                {
                      LastException = exceptionDispatchInfo.SourceException;
                }
            }
```

Reply:
Thank you for view comment
I modified the 3 methods you suggested.


While referring to other methods for writing unit tests, I saw the following method.
The code below tests the callback, so how about modify this part?